client/pages/about-us/about-us.js

```diff
@@ -1,5 +1,5 @@
-import HeaderComponent from "../../components/header-component.js";
-import FooterComponent from "../../components/footer-component.js";
+import HeaderComponent from "../../components/header.js";
+import FooterComponent from "../../components/footer.js";
```

Comment:
Incorrectly changed. No idea why

Reply:
I saw you fixed and I fixed in footer and i added ``FooterComponent``  to homepage